docs/docs/upgrading/migrate_v5_v6.md

```diff
@@ -0,0 +1,42 @@
+---
+sidebar_position: 1
+---
+
+# Upgrading to ICS v6.x from v5.x
```

Comment:
I'm not that familiar with the `upgrading` files. But, do we need any specific migrations for to the work performed in this PR? It doesn't seem to me this file adds information on how to migrate.